src/components/InspirationSection.vue

```diff
@@ -5,68 +5,43 @@
-                Plan's name
-            </h3>
-
-            <the-exercise
-                img="/img/barbell_squat.gif"
-                name="Barbell full squat"
-                muscle="Legs"
-                sets="3"
-                reps="5"
-                weight="45 kg"
-                rest="01:00"
-            />
-            <the-exercise
-                img="/img/b_hip_thrust.gif"
-                name="Barbell hip thrust"
-                muscle="Glutes, Legs"
-                sets="3"
-                reps="8"
-                weight="85 kg"
-                rest="01:00"
-            />
-            <the-exercise
-                img="/img/cable_pullldown.gif"
-                name="Cable pulldown"
-                muscle="Back"
-                sets="3"
-                reps="12"
-                weight="25 kg"
-                rest="01:00"
-            />
-            <the-exercise
-                img="/img/assisted_dips.gif"
-                name="Machine assisted dips"
-                muscle="Chest, Triceps"
-                sets="3"
-                reps="10"
-                weight="25 kg"
-                rest="01:00"
-            />
-            <the-exercise
-                img="/img/calf_raise.gif"
-                name="Calf raises"
-                muscle="Calfs"
-                sets="3"
-                reps="15"
-                weight="75 kg"
-                rest="01:00"
-            />
-            <the-exercise
-                img="/img/ab_wheel.gif"
-                name="Ab-wheel rollout"
-                muscle="Abs"
-                sets="3"
-                reps="10"
-                weight="none"
-                rest="01:00"
-            />
+        <div class="flex justify-between w-full gap-x-10 max-[500px]:gap-6 max-[400px]:gap-4">
+            <img
+              @click="decreaseWorkoutNumber"
+              class="w-8 cursor-pointer max-[500px]:w-5"
+              src="/img/chevrons-left-alt-svgrepo-com.svg"
+              alt="previous workout"
+            >
+            <transition-group name="workout">
+              <the-workout
+                v-for="workout in currentWorkout"
+                :key="workout.workout_name"
+                :title="workout.workout_name">
+                  <the-exercise
+                    v-for="exercise in workout.exercises"
+                    :key="exercise" :img="exercise.img"
+                    :name="exercise.name" :sets="exercise.sets"
+                    :reps="exercise.reps"
+                    :weight="exercise.weight"
+                    :rest="exercise.rest">
+                    <span
+                    v-for="muscle in exercise.muscles"
+                    :key="muscle"
+                    class="px-2 mr-1 last:mr-0 text-xs bg-white-color rounded-full">
+                    {{ muscle }}
+                    </span>
+                  </the-exercise>
```

Comment:
Add a vue 3 carousel component instead.

Reply:
This could be a good thing for a separate task

Reply:
What do you mean by a "separate task"? I'm afraid I've never heard of tasks or I know them under a different name.